static/src/js/routes/Downloads/Downloads.js

```diff
@@ -14,6 +14,8 @@ export const Downloads = ({
 }) => {
   const { path } = match
 
+  // TODO: Lazy load the two components? Since user is expected to go to one of the orders if they are on the download page does not make sense
```

Comment:
Do we want to leave this `TODO` in for this PR?

Reply:
No I'll be removing it wanted to talk about it during the PR review so I wouldn't forget to mention